### PR TITLE
Adding 'datadog_integration' to easily control installed integration.

### PR DESCRIPTION
**Pull request by hush-hush**
Allows user to pin integration version using the `integration` command (see more [here](https://docs.datadoghq.com/agent/guide/integration-management/)).

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,11 @@
 CHANGELOG
 =========
 
+# 3.2.0 /
+
+* [DEPRECATION] Drop support for EOL version of Ansible (2.5)
+- [FEATURE] Add the `datadog_integration resource` to easily control installed integration.
```
Comment (dabcoder) [inline CHANGELOG.md]: `integration` --> `integrations` (plural)?

**Review by dabcoder — APPROVED** [review]
One small nit, otherwise looks good to me.

File: README.md [hunk 2]
```diff
@@ -147,6 +166,48 @@ This example will configure the PostgeSQL check through **autodiscovery**:
 ```
 
 
+Pinning an integration
```
Comment (albertvaka) [inline README.md]: Nit: Pinning -> Upgrading/Installing? You can't use this to pin the version that comes with the agent, or an older one, so I wouldn't say it's "pinning".

File: README.md [hunk 3]
```diff
@@ -147,6 +166,48 @@ This example will configure the PostgeSQL check through **autodiscovery**:
...
+----------------------
+
+**Available for Agent v6.8+**
+
+The `datadog_integration` resource will help you to install specific versions
+of a Datadog integrations. Keep in mind the Agent comes with all the
```
Comment (albertvaka) [inline README.md]:
```suggestion
of Datadog integrations. Keep in mind the Agent comes with all the
```

File: README.md [hunk 4]
```diff
@@ -147,6 +166,48 @@ This example will configure the PostgeSQL check through **autodiscovery**:
...
+  datadog_integration:
+    <name of the integration>:
+      action: <'install' or 'remove'>
+      version: <the version to install>
+```
+
+### Example
+
+This example installs version `1.11.0` of the ElasticSearch integration and
+removing the `postgres` integration.
```
Comment (albertvaka) [inline README.md]:
```suggestion
removes the `postgres` integration (keeping the version bundled with the agent).
```

File: tasks/integration.yml [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+
+- name: Validate integrations actions
+  fail:
+    msg: "Unkown action '{{ item.value.action }}' for integration command ({{ item.key }}). Valid actions are 'install' and 'remove'"
+  when: item.value.action != "install" and item.value.action != "remove"
+  loop: "{{ datadog_integration|dict2items }}"
+
+# Remove Integrations
+
+- name: Removing an integrations (Unix)
```
Comment (albertvaka) [inline tasks/integration.yml]:
```suggestion
- name: Removing integrations (Unix)
```

File: tasks/integration.yml [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+  set_fact:
+    datadog_agent_binary_path: "{{ datadog_agent_binary_path_linux }}"
+  when: ansible_os_family != "Windows"
+
+- name: set agent user for integration commmand (windows)
+  set_fact:
+    integration_command_user: "{{ integration_command_user_windows }}"
+  when: ansible_os_family == "Windows"
+
+- name: set agent agent binaryt path (unix)
```
Comment (albertvaka) [inline tasks/integration.yml]:
```suggestion
- name: set agent agent binary path (unix)
```

File: tasks/integration.yml [hunk 7]
```diff
@@ -0,0 +1,66 @@
...
+      - "{{ datadog_agent_binary_path }}"
+      - integration
+      - remove
+      - "{{ item.key }}"
+  become: yes
+  become_user: "{{ integration_command_user }}"
+  loop: "{{ datadog_integration|dict2items }}"
+  when: item.value.action == "remove" and ansible_os_family != "Windows"
+
+- name: Removing an integrations (Windows)
```
Comment (albertvaka) [inline tasks/integration.yml]:
```suggestion
- name: Removing integrations (Windows)
```

File: tasks/integration.yml [hunk 8]
```diff
@@ -0,0 +1,66 @@
...
+  win_command: "\"{{ datadog_agent_binary_path }}\" integration remove {{ item.key }}"
+  become_user: "{{ integration_command_user }}"
+  loop: "{{ datadog_integration|dict2items }}"
+  when: item.value.action == "remove" and ansible_os_family == "Windows"
+
+# Install integrations
+
+- name: Install pinned version of an integrations (Unix)
```
Comment (albertvaka) [inline tasks/integration.yml]:
```suggestion
- name: Install pinned version of integrations (Unix)
```

File: tasks/integration.yml [hunk 9]
```diff
@@ -0,0 +1,66 @@
...
+      - "{{ datadog_agent_binary_path }}"
+      - integration
+      - install
+      - "{{ item.key }}=={{ item.value.version }}"
+  become: yes
+  become_user: "{{ integration_command_user }}"
+  loop: "{{ datadog_integration|dict2items }}"
+  when: item.value.action == "install" and ansible_os_family != "Windows"
+
+- name: Install pinned version of an integrations (Windows)
```
Comment (albertvaka) [inline tasks/integration.yml]:
```suggestion
- name: Install pinned version of integrations (Windows)
```

File: README.md [hunk 10]
```diff
@@ -9,6 +9,25 @@ this role version 6 of the agent is installed by default (instead of version
 
 Supports most Debian and RHEL-based Linux distributions, and Windows.
 
+- [Ansible Datadog Role](#ansible-datadog-role)
+  * [Installation](#installation)
+  * [Role Variables](#role-variables)
+  * [Agent 5 (older version)](#agent-5--older-version-)
+  * [Dependencies](#dependencies)
+  * [Configuring a check](#configuring-a-check)
+  * [Upgrading an integration](#pinning-an-integration)
```
Comment (albertvaka) [inline README.md]:
```suggestion
  * [Upgrading an integration](#upgrading-an-integration)
```

**Review by albertvaka — APPROVED** [review]
LGTM

**Pull request merged**